.changeset/tiny-gifts-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@internal/analytics-module-segment': major
```

Comment:
With our release flow you'll want to start out the package at version `0.0.0` in this PR, and make this a `minor` bump. That way the first published version will be `0.1.0`

Reply:
ah, makes sense, thanks!